contracts/test/ocr2/aggregator-test.ts

```diff
@@ -188,10 +188,27 @@ describe('aggregator.cairo', function () {
 
     let signatures = []
 
+    console.log([
+      // report_context
+      `0x${config_digest.toString(16)}`,
+      epoch_and_round,
+      extra_hash,
+      // raw_report
+      observation_timestamp,
+      observers,
+      observations.length,
+      ...observations,
+      juels_per_fee_coin,
+    ])
+    console.log(msg)
     for (let oracle of oracles.slice(0, f + 1)) {
       let [r, s] = ec.sign(oracle.signer, msg)
+      console.log(
+        `privKey ${oracle.signer.getPrivate()} r ${r} s ${s} pubKey ${number.toBN(ec.getStarkKey(oracle.signer))}`,
```

Comment:
Is it necessary to log a private key here? 
Seems unsafe even for testing env..

Reply:
hmmm i think this is ok since we use the info from this log statement to test the key signing algorithms - so the logged information is then just hard coded in the go test - so it's already exposed there.

also, i believe it is basically impossible to generate a private key attached to a real account...so generating and logging one-off keys should be ok